go/chat/convsource.go

```diff
@@ -140,12 +143,36 @@ func (s *HybridConversationSource) Pull(ctx context.Context, convID chat1.Conver
 					MsgID:          localData.Messages[0].GetMessageID(),
 				})
 				if err != nil {
-					return chat1.ThreadView{}, nil, err
+					return chat1.ThreadView{}, nil, nil, err
 				}
 				rl = append(rl, res.RateLimit)
 			}
 
-			return localData, rl, nil
+			var breaks []keybase1.TLFUserBreak
```

Comment:
I don't understand why we are doing this, we already identified these people if they are in the cache.